src/maintainer/updating_pkgs.rst

```diff
@@ -218,6 +218,18 @@ package (an example would be to re-render the feedstock to support new Python ve
 
 If you believe a feedstock should be archived, please contact `@conda-forge/core <https://github.com/orgs/conda-forge/teams/core>`__.
 
+Taking over Abandoned Feedstock
+================================
+
+A feedstock is generally considered abandoned if it has no or unresponsive maintainers. Also, it may have multiple bot PRs opened. These feedstocks can be archived or need new maintainers.
+
+To take over the abandoned feedstock and become their new maintainers :
+
+1. Create a `Pull Request <https://conda-forge.org/docs/misc/00_intro.html>`__ to the feedstock repository and add your username to the ``meta.yaml``. (Make sure not to remove the current maintainer’s name from it unless requested by them)
```

Comment:
Need to add `[ci skip] [skip ci] ***NO_CI***` to commit message and pull request title.

Reply:
Added. Anything else on this? Please let me know. @isuruf